docs/MigrationGuide.md

```diff
@@ -5,6 +5,7 @@
 - [Migration Guide](#migration-guide)
   - [To v0.2+](#to-v02)
   - [To v0.3+](#to-v03)
+    - [To v0.3.1](#to-v031)
```

Comment:
It is a bug fix and not a feature?

Reply:
No minor (https://semver.org/) as it looks more like a new feature: v0.4.0

Reply:
8a4ee1a